frontend/src/stores/hydrologic.js

```diff
@@ -417,19 +493,23 @@ export const useHydrologicStore = defineStore('hydrologic', () => {
       default: false,
       short_definition:
         'The maximum number of SWOT passes to intersect each reach during the 21 day orbit cycle',
+      swotviz_alias: 'SWOT Pass Count',
       units: '',
-      plottable: false
+      plottable: false,
+      significant_figures: 0 
     },
     {
-      abbreviation: 'swot_orbits',
+      abbreviation: 'swot_orbit',
```

Comment:
![image](https://github.com/user-attachments/assets/d8fb95ec-9f83-41f6-98c0-b60c504ea737)
do we want this to be delimited in any way? (just for display)

Reply:
That's a great idea!

Reply:
Done!